docs/getting-started-guides/aws.md

```diff
@@ -2,12 +2,31 @@
 assignees:
 - justinsb
 - lavalamp
+- clove
 
 ---
 
 * TOC
 {:toc}
 
+
+## Supported Production Grade Tools with High Availability Options
+
+* [Kubernetes Operations](https://github.com/kubernetes/kops) - Production Grade K8s Installation, Upgrades, and Management. Supports running Debian and CentOS in AWS.
```

Comment:
Releases?  I don't want to maintain that 😸   I would do just distros.  The install provides the ami anyways.